app/src/lib/databases/repositories-database.ts

```diff
@@ -23,6 +23,12 @@ export interface IDatabaseGitHubRepository {
   readonly lastPruneDate: number | null
 }
 
+export interface IDatabaseGitHubBranch {
+  readonly repoId: number
+  readonly name: string
+  readonly protected: boolean
```

Comment:
For the moment we're just looking at the `protected` column because that was easy to compute, but there might come a day down the track where we store more information in this table. In this case I'd recommend just dropping and recreating the table with a newer schema.